README.md

```diff
@@ -18,13 +18,13 @@
 
 ## Why is this so awesome?
 
-* **🚀 Integration with other Nextcloud apps!** Currently Contacts, Calendar & Files – more to come.
+* **🚀 Integration with other Nextcloud apps!** Currently Contacts, Calendar, Files & Tasks – more to come.
 * **📥 Multiple mail accounts!** Personal and company account? No problem, and a nice unified inbox. Connect any IMAP account.
-* **🔒 Send & receive encrypted mails!** Using the great [Mailvelope](https://mailvelope.com) browser extension.
+* **🔒 Send & receive encrypted mails!** Using the great [Mailvelope](https://mailvelope.com) browser extension or the built-in support for S/MIME encryption and signatures.
 * **📑 Message threads!** Now we have proper grouping of message threads.
 * **🗄️ Mailbox management!** You can edit, delete, add submailboxes and more.
 * **🙈 We’re not reinventing the wheel!** Based on the great [Horde](https://www.horde.org) libraries.
-* **📬 Want to host your own mail server?** We don’t have to reimplement this as you could set up [Mail-in-a-Box](https://mailinabox.email)!
+* **📬 Want to host your own mail server?** We don’t have to reimplement this as you could set up [Mail-in-a-Box](https://mailinabox.email) or [Stalwart](https://stalw.art)!
```

Comment:
I would also add Dovecot, as they are a partner and we can developing deeper integration for it.

Reply:
Done